### ISSUE TITLE
path.py Version incorrect in PKG-INFO

### ISSUE THREAD

**Issue by waveman68** [jaraco/path.py#112]
Currently the path.py PKG-INFO is:
```
Metadata-Version: 1.1
Name: path.py
Version: 0.0.0
...
```
This results in the path.py showing up twice in the PyCharm package manager (once as 8.2 and once as 0.0.0). This is corrected by:
```
Metadata-Version: 1.1
Name: path.py
Version: 8.2
...
```

**Comment by jaraco** [comment]
The problem appears to be a bug or incorrect usage with the Travis CI Python version detection. The [relevant build](https://travis-ci.org/jaraco/path.py/builds/121680278) has a job for several Python versions, and each version is attempting a deploy. I'm guessing that Python 2.6 won and did so with an old, unsupported version of setuptools.

**Comment by jaraco** [comment]
Thanks for the report. I've confirmed it's fixed with 8.2.1. Let me know if you find otherwise.
